config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleEntityBuilder.java

```diff
@@ -52,13 +52,18 @@
 
     public Map<String, BundleArtifacts> build(Bundle bundle, BundleType bundleType,
                                               Document document, ProjectInfo projectInfo) {
+        return build(bundle, bundleType, document, projectInfo, false);
+    }
+
+    public Map<String, BundleArtifacts> build(Bundle bundle, BundleType bundleType,
+                                              Document document, ProjectInfo projectInfo, boolean generateMetadata) {
```

Comment:
Why is `generateMetadata` is required? Is there a case when we don't want to generate metadata but create Env bundle?

Reply:
this same method is called by Full bundle task for creating environment entities